host/xtest/regression_2000.c

```diff
@@ -511,11 +507,13 @@ static void *xtest_tee_test_2002_thread(void *arg)
 
 static void xtest_tee_test_2002(ADBG_Case_t *c)
 {
-	pthread_t thr[NUM_THREADS];
 	struct test_2002_barrier bar = { .mu = PTHREAD_MUTEX_INITIALIZER };
-	struct test_2002_arg arg[NUM_THREADS];
-	size_t n;
-	size_t nt;
+	struct test_2002_arg arg[NUM_THREADS] = { };
+	size_t n = 0;
+	size_t nt = 0;
+	pthread_t thr[NUM_THREADS];
```

Comment:
I wonder if this couldn't be initialized with `{ }`, given that it's an array.

Reply:
good suggestion. I'll do.